omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java

```diff
@@ -355,10 +355,7 @@ public String post(@RequestParam(value = "username", required = false) String us
 	 * @return
 	 */
 	private boolean isSelectLocationRequest(boolean isUserLocPropEnabled) {
-		if (isUserLocPropEnabled && Context.isAuthenticated() && Context.getUserContext().getLocationId() == null) {
-			return true;
-		}
-		return false;
+		return (isUserLocPropEnabled && Context.isAuthenticated() && Context.getUserContext().getLocationId() == null);
```

Comment:
Why not?
```suggestion
		return isUserLocPropEnabled && Context.isAuthenticated() && Context.getUserContext().getLocationId() == null;
```

Reply:
Have addressed the comment and updated the PR